go-controller/pkg/ovn/master_test.go

```diff
@@ -235,7 +235,7 @@ var _ = Describe("Master Operations", func() {
 			Expect(fexec.CalledMatchesExpected()).To(BeTrue())
 			updatedNode, err := fakeClient.CoreV1().Nodes().Get(nodeName, metav1.GetOptions{})
 			Expect(err).NotTo(HaveOccurred())
-			Expect(updatedNode.Annotations).To(HaveKeyWithValue(OvnHostSubnet, nodeSubnet))
+			Expect(updatedNode.Annotations).To(HaveKeyWithValue(OvnNodeSubnets, fmt.Sprintf(`{"default": "%s"}`, nodeSubnet)))
```

Comment:
@girishmg can you use MatchJSON() here instead? That ignores formatting/whitespace differences.

Reply:
will do